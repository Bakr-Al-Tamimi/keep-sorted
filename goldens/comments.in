Sticky comments:
// keep-sorted-test start sticky_comments=yes
foo
// comment
bar
// multi line
// comment
baz
// keep-sorted-test end

Sticky comments with different comment sequence:
# keep-sorted-test start sticky_comments=yes
foo
# comment
bar
# keep-sorted-test end

Sticky comments with wrong comment sequence:
# keep-sorted-test start sticky_comments=yes
foo
bar
// comment with wrong comment sequence
# keep-sorted-test end

Trailing comments:
// keep-sorted-test start sticky_comments=yes
foo
// comment
bar
// trailing
// comment
// keep-sorted-test end

Only sticky comments:
// keep-sorted-test start sticky_comments=yes
// only
// comments
// keep-sorted-test end

Only non-sticky comments:
// keep-sorted-test start sticky_comments=no
// only
// comments
// keep-sorted-test end
