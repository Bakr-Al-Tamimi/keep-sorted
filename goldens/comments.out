Sticky comments:
// keep-sorted-test start sticky_comments=yes
// comment
bar
// multi line
// comment
baz
foo
// keep-sorted-test end

Sticky comments with different comment sequence:
# keep-sorted-test start sticky_comments=yes
# comment
bar
foo
# keep-sorted-test end

Sticky comments with wrong comment sequence:
# keep-sorted-test start sticky_comments=yes
// comment with wrong comment sequence
bar
foo
# keep-sorted-test end

Trailing comments:
// keep-sorted-test start sticky_comments=yes
// comment
bar
foo
// trailing
// comment
// keep-sorted-test end

Only sticky comments:
// keep-sorted-test start sticky_comments=yes
// only
// comments
// keep-sorted-test end

Only non-sticky comments:
// keep-sorted-test start sticky_comments=no
// comments
// only
// keep-sorted-test end
